includes/init/init_web.php

```diff
@@ -53,7 +53,7 @@
         'domain'   => $site_url_parts['host'],      // and only for the precise domain
         'secure'   => $isSecure,                    // It uses secure mode if ssl forced or in use
         'httponly' => true,                         // not reachable through javascript
-        'samesite' => 'Strict'                      // strict cookie settings
```

Comment:
I see this in the 3.0 PR but not in the 2.0 version of this update. The `development` branch uses strict here as well (https://github.com/filesender/filesender/blob/80a942408574e4ef804a2dd0e89e3132a62fc2ee/includes/init/init_web.php#L56) is there something in the 3.x code that requires this change for the OIDC?